src/Transition/CreateNewIDFUsingRulesV8_6_0.f90

```diff
@@ -728,6 +727,36 @@ SUBROUTINE CreateNewIDFUsingRules(EndOfFile,DiffOnly,InLfn,AskForInput,InputFile
                   OutArgs(1)='*'
                   nodiff=.false.
                 ENDIF
+
+                ! For Output:Variable that reference a specific Daylighting:Controls object need update to the new reference point name
+                IF (InArgs(1) .NE. '*') THEN
+                  isDElightOutVar = .FALSE. ! first assume that it is not a DElight related variable
+                  IF (SameString(InArgs(2)(1:27),'Daylighting Reference Point')) THEN
```

Comment:
@JasonGlazer I'm not seeing code to convert from the delight "Daylighting Reference Point Illuminance" to "Daylighting Reference Point 1 Illuminance", "Daylighting Reference Point 2 Illuminance", etc.
@Myoldmopar You sure the extra test file gets fully transitioned? DElightCFSLightShelf-TestAllOutputs.idf [attached here](https://github.com/NREL/EnergyPlus/pull/5826#issuecomment-248374806)